src/Entity/ClickNCollectShipment.php

```diff
@@ -21,20 +21,14 @@
  */
 trait ClickNCollectShipment
 {
-    /**
-     * @ORM\ManyToOne(targetEntity=\CoopTilleuls\SyliusClickNCollectPlugin\Entity\LocationInterface::class)
-     */
+    #[ORM\ManyToOne(targetEntity: LocationInterface::class)]
```

Comment:
`targetEntity` and `type` properties should be optional when using attributes. The code can probably be simplified.

Reply:
Done

Reply:
You can remove similar props (such as `type` from all attributes.